modules/reference/profiles/laptop-x86.nix

```diff
@@ -107,9 +97,11 @@ in
         };
       };
 
+      # Enable givc
+      givc.enable = config.ghaf.profiles.debug.enable;
```

Comment:
why tied to debug mode?

Reply:
The `config.ghaf.givc.enable` flag is the global flag to enable givc. It does not work in release currently, as we have no host connection / -debug network. The envisioned solution for this is to introduce admin - host communication over an individual socket, instead of the VM network.